composer.json

```diff
@@ -36,7 +36,7 @@
         }
     },
     "scripts": {
-        "test": "SYMFONY_PHPUNIT_REMOVE_RETURN_TYPEHINT=1 vendor/bin/simple-phpunit",
+        "test": "vendor/bin/simple-phpunit",
```

Comment:
Isn't that needed for you to run the tests on latest phpunit? Like when using PHP 8 IIRC I needed this.

Reply:
I refactored the tests in https://github.com/composer/xdebug-handler/pull/102 to work with PHPUnit8 and got round the type-hinted methods by replacing them with `@before`. `@beforeClass` etc annotated methods. So there is no need for `SYMFONY_PHPUNIT_REMOVE_RETURN_TYPEHINT`.

The tests run fine in PHP-8: https://github.com/composer/xdebug-handler/runs/2309173611?check_suite_focus=true

Reply:
I see, interesting workaround :)